verilog/analysis/checkers/enum_name_style_rule.cc

```diff
@@ -81,6 +100,19 @@ void EnumNameStyleRule::HandleSymbol(const verible::Symbol& symbol,
   }
 }
 
+absl::Status EnumNameStyleRule::Configure(absl::string_view configuration) {
+  using verible::config::SetRegex;
+  absl::Status s = verible::ParseNameValues(
+      configuration, {{"style_regex", SetRegex(&style_regex_)}});
+  if (!s.ok()) return s;
+
+  violation_message_ =
```

Comment:
I am wondering: instead of having a validation_message_ member that we most likely never use, yet allocate and prepare should we maybe have a `std::string CreateValidationMessage() {}` that creates it on-the-fly if needed in the actual valdation ? It also would mean that we don't have to duplicate the code creating it in the constructor and in the `Configure()` method.

Reply:
No problem.